vtex/toolbelt
# Cmd logs

#### What is the purpose of this pull request?
Adds a brand new command: `vtex logs`! This allows developers listen the logs from the VTEX IO apps they created (currently only for non-VTEX accounts).

#### What problem is this solving?
This increases visibility of what happened in their IO apps, helping to visualize the errors this app is throwing in production environments.

#### How should this be manually tested?
1. Switch to an account that uses our logger in their IO apps (`tbb` for example) or use `storecomponents` (you'll probably need to link some application with logs and change the vendor to `storecomponents`).
2. Use `vtex logs --all` and/or `vtex logs --all`
3. Use `vtex logs chaordic-graphql`

If no logs are showing, all logs may already been consumed. Use `vtex logs chaordic-graphql --past` to see logs already seen by someone.

You can also run `vtex logs` in the root path of an IO app and the filter will be read from the manifest.json

#### Screenshots or example usage

#### Types of changes

#### Chores checklist
